nomos-da/kzgrs-backend/benches/verifier.rs

```diff
@@ -26,7 +26,7 @@ fn verify<const SIZE: usize>(bencher: Bencher, column_size: usize) {
 
             let encoder = DaEncoder::new(params);
             let data = rand_data(SIZE * MB / DaEncoderParams::MAX_BLS12_381_ENCODING_CHUNK_SIZE);
-            let encoded_data = encoder.encode(&data).unwrap();
+            let encoded_data = nomos_core::da::DaEncoder::encode(&encoder, &data).unwrap();
```

Comment:
Why this change?

Reply:
I think I had a rust-analyzer bit laggy, because it didn't let recognize the `encode` method from the DaEncoder even after the import. Tried now and it works, fixed.